docs/integrations/github/discovery.md

```diff
@@ -38,17 +38,28 @@ The target is composed of three parts:
 
 ## GitHub API Rate Limits
 
-GitHub
-[rate limits](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting)
-API requests to 5,000 per hour (or more for Enterprise accounts). The default
-Backstage catalog backend refreshes data every 100 seconds, which issues an API
-request for each discovered location.
+GitHub [rate limits] API requests to 5,000 per hour (or more for Enterprise
+accounts). The default Backstage catalog backend refreshes data every 100
+seconds, which issues an API request for each discovered location.
 
 This means if you have more than ~140 catalog entities, you may get throttled by
 rate limiting. This will soon be resolved once catalog refreshes make use of
 ETags; to work around this in the meantime, you can change the refresh rate of
-the catalog in your `packages/backend/src/plugins/catalog.ts` file, or configure
-Backstage to use the [github-apps plugin](../../plugins/github-apps.md).
+the catalog in your `packages/backend/src/plugins/catalog.ts` file:
+
+```typescript
+const builder = await CatalogBuilder.create(env);
+
+// For example, to refresh every 5 minutes (300 seconds).
+builder.setRefreshIntervalSeconds(300);
```

Comment:
The `.setRefreshIntervalSeconds()` actually returns the builder back so the object needs to be stored or used right away.

This line is perhaps better so that readers don't make that mistake `await builder.setRefreshIntervalSeconds(300).build()`

Reply:
It seems to return a reference to the same object either way (`return this`), not a new object, so they should be functionally equivalent. ([link](https://github.com/backstage/backstage/blob/master/plugins/catalog-backend/src/next/NextCatalogBuilder.ts#L153))

Reply:
I'm OK with either, at this point we probably can't change that behaviour of the builder (making it immutable or whatnot).